benchmark-caliper/build.gradle

```diff
@@ -0,0 +1,142 @@
+buildscript {
```

Comment:
Why do we need `benchmark-caliper` at all? I don't think we'll ever want to use caliper on non-android .. JMH is the preferred framework.